src/js/constants/StatusIcon.js

```diff
@@ -0,0 +1,20 @@
+const STATUS_ICON = {
```

Comment:
Even though a couple of naming here (STOPPED, WARNING) might have a coincidence with service status, this is not to match what we have there but the name was given because of the action of the icon.